istioctl/cmd/proxyconfig.go

```diff
@@ -61,6 +161,42 @@ func setupConfigdumpEnvoyConfigWriter(podName, podNamespace string, out io.Write
 	return cw, nil
 }
 
+func setupEnvoyLogConfig(param, podName, podNamespace string) (string, error) {
+	kubeClient, err := clientExecFactory(kubeconfig, configContext)
+	if err != nil {
+		return "", fmt.Errorf("failed to create k8s client: %v", err)
```

Comment:
https://istio.io/about/contribute/style-guide/#miscellaneous

Use `Kubernetes` not `kubernetes` or `k8s`